src/python_inspector/resolution.py

```diff
@@ -299,11 +300,16 @@ def get_requirements_from_python_manifest(
             )
 
         else:
-            # We should not raise exception here as we may have a setup.py that does not
-            # have any dependencies. We should not fail in this case.
-            raise Exception(
-                f"Unable to collect setup.py dependencies securely: {setup_py_location}"
-            )
+            # Do not raise exception here as we may have a setup.py that does not
+            # have any dependencies.
+            with (open(setup_py_location)) as sf:
```

Comment:
Could it make sense to parse the AST instead? Or this is because using the AST did not work in the first place?

Reply:
done

Reply:
Thanks!